exporter/actions.py

```diff
@@ -24,6 +24,11 @@ def __init__(self, build_path):
         self.build_path = build_path
 
     def transform_content(self, obj, raw_content):
+        """
+        Transforms the content returned by the API into something that the frontend expects.
+        This is because there are some differences between the frontend REST handler and the filesystem one
+        (e.g. images with different formats etc.)
+        """
```

Comment:
are you sure you won't be serializing dates? maybe use django default serializer `import json from django.core.serializers`?